src/shared/utils/index.ts

```diff
@@ -204,3 +204,51 @@ export function blacklistKeys(raw: { [key: string]: any }, keys: string[]) {
       return obj;
     }, {});
 }
+
+/**
+ * True if given date is today, false otherwise
+ */
+export function isToday(date: string): boolean {
+  const today = new Date(Date.now());
+  const givenDate = new Date(date);
+
+  if (givenDate.toString() === 'Invalid Date') {
+    return false;
+  }
+  return (
+    givenDate.getDay() === today.getDay() &&
+    givenDate.getMonth() === today.getMonth() &&
+    givenDate.getFullYear() === today.getFullYear()
+  );
+}
+
+/**
+ * How long ago from today in days
+ *
+ * return -1 if given date if from the future
+ *
+ */
+export function howLongAgo(date: string): number {
+  const today = new Date(Date.now());
+  const givenDate = new Date(date);
+  const millisecondsInDay = 24 * 60 * 60 * 1000;
```

Comment:
Comparing dates can be tricky (with TZ, leap seconds, DST, etc). Also, we already have momentjs (or a similar lib) in the indirect dependencies, why not use it here to compare date cleanly?